theoplayer/getting-started/01-sdks/02-android/00-getting-started.mdx

```diff
@@ -0,0 +1,476 @@
+We need to call the `onResume`, `onPause` and `onDestroy` methods of the `THEOplayerView` when the matching methods are called in the Activity.
+
+<Tabs queryString="lang">
+       <!-- prettier-ignore-start -->
+       <TabItem value="kotlin" label="Kotlin">
+        ```kotlin
+        class MainActivity : ComponentActivity() {
+
+            // ...
+
+            override fun onPause() {
+                super.onPause()
+                theoPlayerView.onPause()
+            }
+
+            override fun onResume() {
+                super.onResume()
+                theoPlayerView.onResume()
+            }
+
+            override fun onDestroy() {
+                super.onDestroy()
+                theoPlayerView.onDestroy()
+            }
+
+            // ...
+
+        }
+        ```
+        </TabItem>
+        <TabItem value="java" label="Java">
+        ```java
+        public class MainActivity extends AppCompatActivity{
+
+            // ...
+
+            @Override
+            protected void onPause() {
+                super.onPause();
+                theoPlayerView.onPause();
+            }
+
+            @Override
+            protected void onResume() {
+                super.onResume();
+                theoPlayerView.onResume();
+            }
+
+            @Override
+            protected void onDestroy() {
+                super.onDestroy();
+                theoPlayerView.onDestroy();
+            }
+
+            // ...
+        }
+        ```
+        </TabItem>
+        <!-- prettier-ignore-end -->
+    </Tabs>
+
+If you would like to customize the Fullscreen Activity, you can take a look at our sample [here](https://github.com/THEOplayer/samples-android-sdk/tree/master/full-screen-handling).
+
+## Run your application
+
+Here you have some screenshots from the running application:
+
+### Start screen
+
+![](../../../assets/img/android-getting-started-screenshot-1.png)
+
+### Playing screen
+
+![](../../../assets/img/android-getting-started-screenshot-2.png)
+
+### Paused screen
+
+![](../../../assets/img/android-getting-started-screenshot-3.png)
+
+### Rotated fullscreen
+
+![](../../../assets/img/android-getting-started-screenshot-4.png)
+
+## Summary
+
+This guide focuses on creating an Android application from scratch and integrating the THEOplayer Android SDK.
+
+- You have learned how to interact with the player via the available APIs in Java,
+- you have registered player events,
+- and you have learned how to enable automatic orientation detection.
+
+You can find various samples for different use cases on [GitHub](https://github.com/THEOplayer/samples-android-sdk/tree/master).
+
+You may also find the readme at [https://github.com/THEOplayer/theoplayer-sdk-android](https://github.com/THEOplayer/theoplayer-sdk-android) useful.
+
```

Comment:
I was hoping we could move all of the useful information from that readme into our documentation website, so we would no longer need to link to it.

Reply:
I checked that readme and I think the all useful info from there is already here. So, indeed, we don't need to include this link.